webapp/components/survey/NodeDefDetails/store/actions/useCancelEdits.js

```diff
@@ -7,30 +7,44 @@ import { NodeDefsActions } from '@webapp/store/survey'
 
 import { State } from '../state'
 
-export const useCancelEdits = () => {
+export const useCancelEdits = ({ setState }) => {
   const history = useHistory()
   const dispatch = useDispatch()
 
   const cancelEdits = ({ state }) => async () => {
     const nodeDef = State.getNodeDef(state)
     const nodeDefOriginal = State.getNodeDefOriginal(state)
 
+    await setState(State.assocNodeDef(nodeDefOriginal))
```

Comment:
do we like to restore to the original node def? or just remain the node dirty?

Reply:
this is necessary to "disable" the interception of back button press when user presses the Cancel button and history.back() is called